build.sbt

```diff
@@ -70,7 +70,7 @@ val commonSettings = Seq(
   scalacOptions ++= Seq(
     "-feature",
     "-deprecation",
-    "-unchecked",
-    "-Xlint"
+//    "-Xlint",
```

Comment:
We should probably at least just ignore the warning that is causing the problem but allow the other warnings to come through.